src/index.html

```diff
@@ -22,6 +22,335 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__item">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="/"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="icon"
+        />
+
+        <div class="item">
+          <div class="item__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="item__code">Product code: 195434</div>
+          <div class="reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="reviews__number">Reviews: 5</div>
+          </div>
+        </div>
+        <div class="price">
+          <div class="price__name">Price:</div>
+          <div class="price__value">$2,199</div>
+        </div>
+        <a
+          data-qa="card-hover"
+          class="btn"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```

Reply:
Not fixed